datafusion/physical-expr/src/equivalence/class.rs

```diff
@@ -851,6 +851,26 @@ impl EquivalenceGroup {
             .zip(right_children)
             .all(|(left_child, right_child)| self.exprs_equal(left_child, right_child))
     }
+
+    /// Determine whether fields at indices `lhs` and `rhs` are equal.
+    pub fn fields_equal(&self, lhs: usize, rhs: usize) -> bool {
```

Comment:
I'm little bit struggling to understand what kind of indices are `lhs` and `rhs`, this is a `pub` method, should we explain more in comments what those indices are?